src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -404,11 +405,14 @@ function MoneyRequestView({
                             if (!transaction?.transactionID) {
                                 return;
                             }
-                            if (
-                                transaction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD &&
-                                Object.values(transaction?.errors ?? {})?.find((error) => ErrorUtils.isReceiptError(error))
-                            ) {
-                                deleteTransaction(parentReport, parentReportAction);
+                            if (transaction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD) {
```

Comment:
Let us do an early return here to keep possible regressions at bay. Also, I think doing any further steps does not make sense when the chat report and all related reports are deleted. Something like this can be done within the `if` condition here. What do you think?

```
if (ReportUtils.getAddWorkspaceRoomOrChatReportErrors(chatReportID)) {
   Report.navigateToConciergeChatAndDeleteReport(chatReportID, true);
   return;
}
if (Object.values(transaction?.errors ?? {})?.find((error) => ErrorUtils.isReceiptError(error))) {
   deleteTransaction(parentReport, parentReportAction);
}
```

Reply:
That makes sense, updated!